fleet/lib/rook-ceph-conf/charts/gaw/templates/obc-yagan-k8up.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
+metadata:
+  name: &name yagan-k8up
+  namespace: rook-ceph
+spec:
+  bucketName: *name
+  storageClassName: k8up-backups
+  additionalConfig:
+    bucketOwner: yagan-k8up
+    bucketMaxSize: 10Ti
```

Comment:
yagan is much larger, I would increase bucket size according to current yagan use.

Reply:
Yeah, i agree its on DRAFT at the moment. I had some questions regards this that I wanted to address with you because we are talking big amounts of data.